services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentController.kt

```diff
@@ -182,4 +211,10 @@ class ApplicationAttachmentController(
     @Hidden
     fun applicationNotFound(ex: ApplicationNotFoundException): HankeError =
         HankeError.HAI2001.also { logger.warn(ex) { ex.message } }
+
+    @ExceptionHandler(ApplicationAlreadyProcessingException::class)
+    @ResponseStatus(HttpStatus.CONFLICT)
+    @Hidden
+    fun applicationAlreadyProcessing(ex: ApplicationAlreadyProcessingException): HankeError =
```

Comment:
This and the not found handler could also be moved to ControllerExceptionHandler, so they don't have to be duplicated here and in ApplicationController.

Reply:
Changed as suggested.